packages/sdk/src/index.ts

```diff
@@ -117,4 +117,4 @@ export type { Item as CartItem } from './cart/Cart'
 export type { Cart } from './cart/Optimistic'
 
 // Storage
-export { useStorage } from './storage/useStorage'
+export { useStorage } from './storage/useStorage'
```

Comment:
A little thing: Github is complaining about new line.
<img width="514" alt="Screen Shot 2022-07-08 at 19 11 51" src="https://user-images.githubusercontent.com/11325562/178076670-621aba44-cf01-4e44-be39-fb1f210cecf7.png">